### PR TITLE
feat(action): Add Docker action

### PR DESCRIPTION
**Pull request by simar7**
This PR adds the ability to run Docker containers as actions.

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

File: outputs/docker.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		return fmt.Errorf("docker action failed to pull docker image: %w", err)
+	}
+
+	var hc container.HostConfig
+	if len(d.Volumes) > 0 {
+		for src, dst := range d.Volumes {
+			hc.Mounts = append(hc.Mounts, mount.Mount{Type: mount.TypeBind, Source: src, Target: dst})
+		}
+	}
+	d.Env = append(d.Env, fmt.Sprintf("POSTEE_EVENT=%s", m["description"]))
```
Comment (AndreyLevchenko) [inline outputs/docker.go:59]: I didn't find where it's cleared. It may be required if action is invoked more than once

File: outputs/docker.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	d.uuidNew = uuid.New
+
+	log.Println("docker action successfully initialized")
+	return nil
+}
+
+func (d DockerClient) Send(m map[string]string) error {
+	ctx := context.Background()
+
+	_, err := d.client.ImagePull(ctx, d.ImageName, types.ImagePullOptions{})
```
Comment (AndreyLevchenko) [inline outputs/docker.go:48]: ignored argument is `io.ReadCloser` instance which needs to be closed explicitily

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: outputs/docker.go (hunk 1 above)
Comment (simar7) [reply outputs/docker.go]: thanks - fixed https://github.com/aquasecurity/postee/pull/312/commits/e39af060ab40098ce57ee55460ab8504045acbb1

File: outputs/docker.go (hunk 2 above)
Comment (simar7) [reply outputs/docker.go]: thanks, good catch - fixed here https://github.com/aquasecurity/postee/pull/312/commits/18b6760015d8d91248985c9268cc57cfbada1d08

**Review by AndreyLevchenko — APPROVED** [review]

**Pull request merged**